### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by hugormdty** [GabrieleGiuseppini/Floating-Sandbox#89]
so i have blackscreen is sometime showing the game itself then go back to black how do to fix?
![image](https://github.com/user-attachments/assets/3bb0ef80-9c8b-4caf-bcd0-87226bfe5058)

**Comment by GabrieleGiuseppini** [comment]
Hello, thank you for your report. This is totally new, never heard of that before.

Do you know what graphics card you have? 

Also, under "C:\Users\<your username>\AppData\Roaming\Floating Sandbox\Diagnostics" there is a "last_run.log" log file. Could you send me that one?

**Comment by hugormdty** [comment]
Yes I can, my graphics card is AMD Radeon Vega 8 graphics.
and I dont se diagnostic file in the game.

**Comment by Ezeril** [comment]
Yes, I have the same problem, I updated my drivers and nothing to do, I remain on a black screen even after uninstalling and reinstalling the game

**Comment by GabrieleGiuseppini** [comment]
> Yes, I have the same problem, I updated my drivers and nothing to do, I remain on a black screen even after uninstalling and reinstalling the game

What's your graphics card?
